test/util.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+# This utility file contains functions that wrap commands to be tested and allow us to interface 
+# our command-line tests with JUnit on Jenkins. All wrapper functions run commands in a sub-shell 
+# and redirect all output. Tests in test-cmd *must* use these functions for testing.
+
+# HIGH-LEVEL WRAPPER FUNCTIONS: ASSUMING EXIT CODES
+
+# expect_success runs the cmd and expects an exit code of 0
+function os::test::expect_success() {
```

Comment:
Also, since the package (file) you are importing (sourcing) from isn't evident in the function description, inside of a script that sources this and uses a function, it *is* of use to know that it's a test wrapper - that information is not given anywhere else.